gcapi/client.py

```diff
@@ -904,3 +781,249 @@ def add_cases_to_reader_study(
 
             res.append(ds.pk)
         return res  # noqa: B901
+
+    def _validate_display_set_values(self, values, interfaces):
```

Comment:
is this meant to be in this PR? It looks it's not getting called, right?

Reply:
Ah, it moved! I originally started with clearing out some helper functions that are not longer needed to paste all of them. Thought it wiser to re-add all those other than the archive-items so I could scope things a bit to keep this PR smallish.

For clarity I'll move it where it was (above add_cases)

Reply:
Ok, so it's needed for some other helper function, but not the ones you're touching here.

Reply:
Indeed!